### PR TITLE
fix the torch.nn.Hardtanh's Examples

### PR DESCRIPTION
**Pull request by AceCoooool**
1. fix the examples in `torch.nn.Hardtanh` 
2. I think there should be an `else` （without it is also right）

### CODE REVIEW

**Review by soumith — APPROVED** [review]

File: torch/lib/THNN/generic/HardTanh.c [hunk 1]
```diff
@@ -26,6 +26,7 @@ void THNN_(HardTanh_updateOutput)(
         else if (*input_data > max_val)
           *input_data = max_val;
       );
+    else
```
Comment (apaszke) [inline torch/lib/THNN/generic/HardTanh.c]: wow, nice catch. Can you add brackets around both branches to make it more explicit? Thanks!

**Review by apaszke — APPROVED** [review]

File: torch/lib/THNN/generic/HardTanh.c (hunk 1 above)
Comment (AceCoooool) [reply torch/lib/THNN/generic/HardTanh.c]: There are two styles of brackets using in pytorch: (I choose the first)
1. else {
2. else
    {

**Review by apaszke — APPROVED** [review]

**Comment by apaszke** [comment]
Thank you!

**Pull request merged**